dns/nixos.org.js

```diff
@@ -171,7 +172,11 @@ D("nixos.org",
 	TXT("_github-pages-challenge-nixos", "f3a423ba6916e972cfb1e74f82f601"),
 
 	// netlify pages
+	ALIAS("@", "nixos-homepage.netlify.app."),
```

Comment:
Apologies if you explained this elsewhere: is this violating the advice from https://answers.netlify.com/t/support-guide-which-are-some-good-dns-providers-for-alias-aname-support/211?

> Some providers we cannot recommend after last review (which is well over 2 years ago at this point) are ALIAS or ANAME records provided by folks like Hover, DNS Made Easy, and even NS1 when not used as a service we provide: those “fancy” record types as implemented at those providers hosting was at last review incompatible with [the way we do GeoIP routing](https://www.ibm.com/products/ns1-connect) for our CDN.

Granted, this post from 2019 sounds a bit hand wavy about a "review" they did in 2017.

Reply:
Yes, I explained that limitation in https://matrix.to/#/!qAqmHNDbGMBTaNCArk:nixos.org/$rOR7m5_eJbT8cJFHfChMLCS20ks0jOL0-c86qiwhqIA?via=nixos.org&via=matrix.org&via=lossy.network, and it is a reason we should be migrating away from netlify going forward.